source/index.js

```diff
@@ -58,15 +58,15 @@ module.exports = async (input = 'patch', options) => {
 	const hasLockFile = fs.existsSync(path.resolve(rootDir, 'package-lock.json')) || fs.existsSync(path.resolve(rootDir, 'npm-shrinkwrap.json'));
 	const isOnGitHub = options.repoUrl && hostedGitInfo.fromUrl(options.repoUrl).type === 'github';
 
-	let published = false;
+	let isPublished = false;
 
 	const rollback = onetime(async () => {
 		console.log('\nPublish failed. Rolling back to the previous state…');
 
-		const lastCommit = await execa.stdout('git', ['log', '-1', '--pretty=%B']);
+		const lastCommit = await execa.stdout('git', ['log', 'HEAD^..HEAD', '--pretty=%B']);
```

Comment:
I meant `--max-count=1` instead of `-1`.

---

```
-<number>, -n <number>, --max-count=<number>
           Limit the number of commits to output.
```